mac/CBMR.MAC

```diff
@@ -1,10 +1,23 @@
+*TODO*
```

Comment:
What's the TODO about?

Reply:
Fixed - not sure what I was thinking there